cluster/common.sh

```diff
@@ -9,7 +9,7 @@ function debug {
 }
 
 function install_cdi {
-    export VERSION=$(curl -s https://github.com/kubevirt/containerized-data-importer/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
```

Comment:
The reason for having that release version dynamic was to detect early that there is incompatibility or bug that needs to be fixed on either side. I wouldn't want to loose that capability. 
Something like following one-liner should be fine (I haven't verified whether it works in all situations/input).
```
curl -s https://api.github.com/repos/kubevirt/containerized-data-importer/releases | jq 'sort_by(.tag_name) | .[-1].tag_name' -r
```